src/backend/distributed/shared_library_init.c

```diff
@@ -939,6 +997,33 @@ RegisterCitusConfigVariables(void)
 		GUC_UNIT_MS,
 		NULL, NULL, NULL);
 
+	DefineCustomIntVariable(
```

Comment:
I really cannot use case for this GUC as well. Can you explain why do you think needed?

Reply:
This is used for testing